src/components/Molecules/DecodedTransaction/DecodedTransaction.tsx

```diff
@@ -262,10 +262,14 @@ export const DecodedTransaction: React.FC<DecodedTransactionProps> = ({
                                                                                 .hex
                                                                         }
                                                                         token_url={
-                                                                            ticker_logo ||
+                                                                            CHAIN?.logo_url ||
```

Comment:
One is SVG and the other is PNG - best to not connect the two.

Reply:
The response from the decoder and API does not return the SVG version. I am speaking with dylan if he can return the logo_urls array in the response